media-libs/flac/flac-1.5.0-r1.ebuild

```diff
@@ -0,0 +1,79 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit flag-o-matic libtool multilib-minimal
+
+DESCRIPTION="Free Lossless Audio Codec (RFC 9639)"
+HOMEPAGE="https://xiph.org/flac/"
+SRC_URI="https://downloads.xiph.org/releases/${PN}/${P}.tar.xz"
```

Comment:
Why are we changing SRC_URI? See a913a35f6a710c6d8f4157a8034b011b6c7d4268 and 97209783eb2728405f7ebd0fc580dd80bfda7009.